src/lang/english.txt

```diff
@@ -1450,11 +1460,17 @@ STR_CONFIG_SETTING_ORDER_REVIEW_EXDEPOT                         :Yes, but exclud
 STR_CONFIG_SETTING_ORDER_REVIEW_ON                              :Of all vehicles
 
 STR_CONFIG_SETTING_WARN_INCOME_LESS                             :Warn if a vehicle's income is negative: {STRING2}
-STR_CONFIG_SETTING_WARN_INCOME_LESS_HELPTEXT                    :When enabled, a news message gets sent when a vehicle has not made any profit within a calendar year
+STR_CONFIG_SETTING_WARN_INCOME_LESS_HELPTEXT                    :When enabled, a news message gets sent when a vehicle has not made any profit within {TKM "a calendar year" "a period"}
```

Comment:
This one still bugs me; why calendar here, and not everywhere else? There might be a good reason, so I am actually asking :D

Reply:
Unnecessarily wordy, I guess. I can make it just `a year`. 😉